XamlStyler.Core/Options/StylerOptions.cs

```diff
@@ -359,7 +366,7 @@ private void InitializeProperties()
             {
                 var assembly = Assembly.GetExecutingAssembly();
                 using (Stream stream = assembly.GetManifestResourceStream(StylerOptions.DefaultOptionsPath))
-                using (StreamReader reader = new StreamReader(stream))
+                using (var reader = new StreamReader(stream))
```

Comment:
Here StreamReader changed to var. Line 358: var changed to string. Do we have sone common style like prefer using var or concrete type declaration?

Reply:
Prefer ```var``` when variable type is apparent, and explicit type otherwise (including built-in types).